config/di.php

```diff
@@ -71,6 +75,19 @@
         return (new SchemaProviderPipeline($container))->withConfig($params['yiisoft/yii-cycle']['schema-providers']);
     },
 
+    // FromFilesSchemaProvider
+    FromFilesSchemaProvider::class => static function (Aliases $aliases) {
+        return new FromFilesSchemaProvider(static fn (string $path): string => $aliases->get($path));
+    },
+
+    // PhpFileSchemaProvider
+    PhpFileSchemaProvider::class => static function (Aliases $aliases, ContainerInterface $container) {
+        return new PhpFileSchemaProvider(
+            static fn (string $path): string => $aliases->get($path),
+            $container->has(FilesInterface::class) ? $container->get(FilesInterface::class) : null
+        );
+    },
```

Comment:
I think we can use this way
https://github.com/yiisoft/definitions?tab=readme-ov-file#references

Reply:
Thanks, added references